src/components/DataDecorator/Decorator/Decorator.js

```diff
@@ -160,6 +171,7 @@ Decorator.defaultProps = {
   href: undefined,
   inline: false,
   onClick: undefined,
+  onContextMenu: () => {},
```

Comment:
@jendowns I went this route to prevent a TypeError when `this.handleContextMenuClick` is called. Alternatively, a check could be add to see if the prop exists if you prefer:
```
onContextMenu={onContextMenu && this.handleContextMenuClick}
```